test/ims.test.js

```diff
@@ -519,3 +519,28 @@ test('Ims.getSusiUrl - callbackUrl null', () => {
       'https://ims-na1.adobelogin.com/ims/authorize/v1?response_type=code&client_id=some-client-id&scope=some%2C+scopes&state=some-state'
     )
 })
+
+test('Ims.getAccessTokenByClientCredentials', async () => {
+  const ims = new Ims()
+
+  const clientId = 'some-client-id'
+  const clientSecret = 'some-client-secret'
+  const orgId = 'some-org-id'
+  const scopes = ['some', 'things']
+
+  const serverResponsePayload = {
+    access_token: '',
+    refresh_token: ''
```

Comment:
there is no refresh_token from a call, shouldn't matter in this case since it's mocked, but for correctness